### PR TITLE
Feature/wallet-addresses-widget

### PR DESCRIPTION
**Pull request by rannyeribaptist**
HI there 👋

This is my first PR for Tokel Project and I updated a few things in the Dashboard:

The history widget is moved up, replacing the graphs widget. I also added shortcuts in the list as suggested by [piggydoughnut](https://github.com/piggydoughnut), so you can copy the **Tx id** and open the transaction in the browser straight from the list:

image

We have a new widget now! The Wallet Addresses widget displays your **account's address** and **pubkey**:

image

The entire Dashboard now looks like this:

image

### CODE REVIEW

File: src/components/Dashboard/widgets/Embeds/ActivityListEmbed.tsx [hunk 1]
```diff
@@ -128,29 +143,34 @@ const ActivityList = ({ transactions = [], resourceType }: ActivityListProps): R
...
-                )} ${TICKER}`}
-                secondary={
-                  resourceType === ResourceType.TOKEL
-                    ? `$${getUsdValue(processPossibleBN(tx.value), tokelPriceUSD)}`
-                    : ''
-                }
-                justify="flex-end"
-              />
-            </Transaction>
+            <ActivityListItem key={tx.txid + tx.received}>
```
Comment (lenilsonjr) [inline src/components/Dashboard/widgets/Embeds/ActivityListEmbed.tsx:146]: we can swap this key to be just the txid

```suggestion
            <ActivityListItem key={tx.txid}>
```

File: src/components/Dashboard/widgets/Embeds/WalletAddressesEmbed.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  title: string;
+  modal_type: string;
+};
+
+const openWalletModal = (options: ReceiveModalOpts) => () =>
+  dispatch.environment.SET_MODAL({ name: ModalName.RECEIVE, options });
+
+const DisplayWalletAddress = ({ title, modal_type }: WalletAddressWidgetProps) => {
+  const tokenInfo = useSelector(selectCurrentTokenInfo);
+  const isNFT = tokenInfo && tokenInfo.supply === 1;
```
Comment (lenilsonjr) [inline src/components/Dashboard/widgets/Embeds/WalletAddressesEmbed.tsx:41]: feel free to use optional chaining here
```suggestion
  const isNFT = tokenInfo?.supply === 1;
```

File: src/components/Dashboard/widgets/Embeds/WalletAddressesEmbed.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        </Button>
+        <Copy>
+          <CopyToClipboard color="white" textToCopy={target} />
+        </Copy>
+      </RowWrapper>
+    </div>
+  );
+};
+
+const WalletAddresses = () => {
```
Comment (lenilsonjr) [inline src/components/Dashboard/widgets/Embeds/WalletAddressesEmbed.tsx:72]: Mind changing this component name to match the filename?

**Review by lenilsonjr — COMMENTED** [review]
looking good! left some notes

**Comment by piggydoughnut** [comment]
Looks good, agree with Lenni's remarks. Its a pass for me. @lenilsonjr said you will look at the tests together. So once the tests pass we can merge 👌🏼

**Comment by piggydoughnut** [comment]
Looks good, agree with Lenni's remarks. Its a pass for me. @lenilsonjr said you will look at the tests together. So once the tests pass we can merge 👌🏼

**Review by piggydoughnut — CHANGES_REQUESTED** [review]
Please apply the changes Lenni suggested and make the tests pass so the CI is happy.

**Review by lenilsonjr — APPROVED** [review]
looks good. Since @piggydoughnut gave green light above, going to merge this one rn

**Pull request merged**